source/slang/reflection.cpp

```diff
@@ -567,7 +567,7 @@ SLANG_API char const* spReflectionType_GetName(SlangReflectionType* inType)
 {
     auto type = convert(inType);
 
-    if( auto declRefType = type->As<DeclRefType>() )
+    if( auto declRefType = as<DeclRefType>(type) )
```

Comment:
This is one of those rare cases where maybe we *shouldn't* be using `as` on a type, but it is fine to leave it as-is since the new code matches the semantics of the old.

The difference here would be if a user looks up a global `typedef` via reflection, and then asks for its name. The current logic will return the name of the type it refers to, while the user might expect us to return the name of the `typedef`.